backend/src/db/cmd.rs

```diff
@@ -206,17 +211,21 @@ fn restore(config: &DbConfig, dump: &Path) -> Result<Never> {
     fork_command(
         Command::new("pg_restore")
             .arg("--dbname")
-            .arg(connection_uri(config))
+            .arg(connection_uri(&DbConfig { database: "postgres".into(), ..config.clone() }))
+            .arg("--clean")
+            .arg("--if-exists")
+            .arg("--create")
```

Comment:
What's this change for? Just asking.

Reply:
That's what I meant in the other comment. `pg_restore` now takes care of dropping (`--clean`) and recreating (`--create`) the db.